.github/workflows/performance.yml

```
@@ -0,0 +1,211 @@
+        
+        echo "Performance badge: $PASS_RATE with color $BADGE_COLOR"
+        # Could integrate with shields.io or similar service
+
+  performance-android:
+    name: Android Performance Tests
+    runs-on: macos-latest
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)